addons/l10n_hk/data/l10n_hk_chart_data.xml

```diff
@@ -0,0 +1,551 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+    <!-- Account Template -->
+    <record id="l10n_hk_1" model="account.account.template">
+        <field name="name">Assets</field>
+        <field name="code">1</field>
+        <field name="reconcile" eval="False" />
+        <field name="user_type_id" ref="account.data_account_type_non_current_assets" />
+        <field name="chart_template_id" ref="l10n_hk_chart_template" />
+    </record>
+    <record id="l10n_hk_11" model="account.account.template">
+        <field name="name">Fixed Assets</field>
+        <field name="code">11</field>
+        <field name="reconcile" eval="False" />
+        <field name="user_type_id" ref="account.data_account_type_non_current_assets" />
+        <field name="chart_template_id" ref="l10n_hk_chart_template" />
+    </record>
+    <record id="l10n_hk_1110" model="account.account.template">
+        <field name="name">Furniture and Fixtures</field>
+        <field name="code">1110</field>
+        <field name="reconcile" eval="False" />
+        <field name="user_type_id" ref="account.data_account_type_non_current_assets" />
+        <field name="chart_template_id" ref="l10n_hk_chart_template" />
+    </record>
+    <record id="l10n_hk_1130" model="account.account.template">
+        <field name="name">Equipment</field>
+        <field name="code">1130</field>
+        <field name="reconcile" eval="False" />
+        <field name="user_type_id" ref="account.data_account_type_non_current_assets" />
+        <field name="chart_template_id" ref="l10n_hk_chart_template" />
+    </record>
+    <record id="l10n_hk_1140" model="account.account.template">
+        <field name="name">Decoration</field>
+        <field name="code">1140</field>
+        <field name="reconcile" eval="False" />
+        <field name="user_type_id" ref="account.data_account_type_non_current_assets" />
+        <field name="chart_template_id" ref="l10n_hk_chart_template" />
+    </record>
+    <record id="l10n_hk_1160" model="account.account.template">
+        <field name="name">Investments</field>
+        <field name="code">1160</field>
+        <field name="reconcile" eval="False" />
+        <field name="user_type_id" ref="account.data_account_type_non_current_assets" />
+        <field name="chart_template_id" ref="l10n_hk_chart_template" />
+    </record>
+    <record id="l10n_hk_12" model="account.account.template">
```

Comment:
I am confused with nowadays flat CoA: why would we need these sub-levels?

Reply:
this account template will create the account 120 000 Current Assets, so yes it is needed ;-)